packages/building-rollup/src/dynamic-import-polyfill.js

```diff
@@ -18,7 +29,7 @@ window.importModule = function importModule(url) {
       URL.revokeObjectURL(script.src);
       script.src = '';
     };
-    script.defer = 'defer';
+    script.defer = true;
```

Comment:
Just to double check, this still produces the correct output?

Reply:
typing said this only accepts true/false and not a string => so it was a type linting finding :)

I did run the demo and it worked fine... but pls verify 🤗